### DISCUSSION TITLE
Issues to deploy kafka-connect

### DISCUSSION THREAD

**Discussion by marncosta** [strimzi/strimzi-kafka-operator#8962] [General]
I am deploying Kafka-connect into a Kubernetes version 1.27.1.
 I am receiving this error: error Kustomization/kafka-operator.flux-system - Reconciliation failed after 179.85045ms, next try in 1h0m0s KafkaConnect/services/kafka-connect dry-run failed: failed to get API group resources: unable to retrieve the complete list of server APIs: [kafka.strimzi.io/v1beta2](http://kafka.strimzi.io/v1beta2): the server could not find the requested resource

**Comment by scholzj** [comment]
You should probably start by sharing more details ... version of Strimzi, logs, steps for reproducing, custom resources used etc. What you shared above seems like a Flux error rather than Strimzi error as it does not contain any Strimzi logs or anything.
